install_config/storage_examples/direct_storage.adoc

```diff
@@ -0,0 +1,481 @@
+= Direct access to storage using two methods
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+In OpenShift a Persistent Volume Claim (PVC) will bind to a Persistent Volume (PV) based on matching the
+spec labels of 'accessModes' and 'capacity' values. In some cases, a user may want to be specific about
+the storage they are acessing. This example will show how to a user can prebind a PV to a specifc claim 
+[METHOD 1] as well as go directly to the storage using the volume plugin [METHOD 2] inside the pod spec.
+
+In  this example we are using NFS, but the same method can be use for any of the storage plugins
+Please see the section on link:../persistent_storage/index.html[Persistent Storage Plugins] for the supports types.
+
+
+[NOTE]
+====
+All `oc ...` commands are executed on the OpenShift master node.
+====
+
+== [METHOD 1] Create the Persistent Volume using a predefined Persistent Volume Claim reference
+=== 1. Creating the persistent volume (PV) 
+The user may define the spec to include a claim name (ClaimRef) to restrict this PV to only
+bind to a particular persistent volume claim.
+
+.Persistent Volume Object Definition Using NFS
+====
+
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs-pv-prebind <1>
+spec:
+  capacity:
+    storage: 1Gi <2>
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain <3>
+  ClaimRef:
+    Namespace: default <4>
+    Name: nfs-claim-prebind <5>
+  nfs: <6>
+    path: /opt/nfs
+    server: nfs.f22
+    readOnly: false
+----
+<1> The name of the PV which is referenced in pod definitions or displayed in
+various `oc` volume commands.
+<2> The amount of storage allocated to this volume.
+<3> A volume reclaim policy of "retain" indicates to preserve the volume after the pods 
+<4> This is the namespace the claim will exist within 
+<5> This is the claim name that the PVC will need to match EXACTLY to in it's definition  
```

Comment:
Not only does the claim name need to match, but also the capacity defined in the PVC must be <= to the PV's capacity and the PVC's accessModes must be compatible with the access label defined in the PV. So all three must match (today).